esx_service/utils/auth_data.py

```diff
@@ -410,7 +422,8 @@ def create_default_tenant(self):
                                            name=auth.DEFAULT_TENANT, 
                                            description="This is a default tenant", 
                                            vms=[], 
-                                           privileges=[])
+                                           privileges=[],
+                                           tenant_uuid=auth.DEFAULT_TENANT_UUID)
```

Comment:
We can deduce the default tenant uuid based on its name. So I'd suggest not to change the logic here. Instead, we can handle this inside create_tenant() method.